apps/www/components/Article/ReadAloudInline.tsx

```diff
@@ -72,18 +73,19 @@ const styles = {
 const SyntheticAudio = ({ meta, t }: { meta: Meta; t: (sting) => string }) => {
   const { toggleAudioPlayer } = useContext<AudioContextType>(AudioContext)
   const [colorScheme] = useColorContext()
-
+  const { kind } = meta.audioSource
+  const Icon = kind === 'syntheticReadAloud' ? AudioIcon : PodcastIcon
```

Comment:
IMO it would be better to not change the event category to avoid a mess in matomo
```suggestion
  const isSynthetic = kind === 'syntheticReadAloud'
  const Icon = isSynthetic ? AudioIcon : PodcastIcon
  const eventCategory = isSynthetic ? 'SyntheticAudio' : 'ReadAloudAudio'
```